pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.609.1</version>
+    <version>2.1</version>
```

Comment:
Yes and it is affected by https://github.com/jenkinsci/jenkins-test-harness/pull/2.

I tested everything locally and it works. I'm closing and reopening (removing the wip part) with the correct parent version and last jth when available.

Thanks!